etl/steps/data/garden/who/2022-09-30/ghe.countries.json

```diff
@@ -0,0 +1,185 @@
+  "DOM": "Dominican Republic",
+  "DZA": "Algeria",
+  "ECU": "Ecuador",
+  "EGY": "Egypt",
+  "ERI": "Eritrea",
+  "ESP": "Spain",
+  "EST": "Estonia",
+  "ETH": "Ethiopia",
+  "FIN": "Finland",
+  "FJI": "Fiji",
+  "FRA": "France",
+  "FSM": "Micronesia (country)",
+  "GAB": "Gabon",
+  "GBR": "United Kingdom",
+  "GEO": "Georgia",
+  "GHA": "Ghana",
+  "GIN": "Guinea",
+  "GMB": "Gambia",
+  "GNB": "Guinea-Bissau",
+  "GNQ": "Equatorial Guinea",
+  "GRC": "Greece",
+  "GRD": "Grenada",
+  "GTM": "Guatemala",
+  "GUY": "Guyana",
+  "HND": "Honduras",
+  "HRV": "Croatia",
+  "HTI": "Haiti",
+  "HUN": "Hungary",
+  "IDN": "Indonesia",
+  "IND": "India",
+  "IRL": "Ireland",
+  "IRN": "Iran",
+  "IRQ": "Iraq",
+  "ISL": "Iceland",
+  "ISR": "Israel",
+  "ITA": "Italy",
+  "JAM": "Jamaica",
+  "JOR": "Jordan",
+  "JPN": "Japan",
+  "KAZ": "Kazakhstan",
+  "KEN": "Kenya",
+  "KGZ": "Kyrgyzstan",
+  "KHM": "Cambodia",
+  "KIR": "Kiribati",
+  "KOR": "South Korea",
+  "KWT": "Kuwait",
+  "LBN": "Lebanon",
+  "LBR": "Liberia",
+  "LBY": "Libya",
+  "LCA": "Saint Lucia",
+  "LKA": "Sri Lanka",
+  "LSO": "Lesotho",
+  "LTU": "Lithuania",
+  "LUX": "Luxembourg",
+  "LVA": "Latvia",
+  "MAR": "Morocco",
+  "MDA": "Moldova",
+  "MDG": "Madagascar",
+  "MDV": "Maldives",
+  "MEX": "Mexico",
+  "MKD": "North Macedonia",
+  "MLI": "Mali",
+  "MLT": "Malta",
+  "MMR": "Myanmar",
+  "MNE": "Montenegro",
+  "MNG": "Mongolia",
+  "MOZ": "Mozambique",
+  "MRT": "Mauritania",
+  "MUS": "Mauritius",
+  "MWI": "Malawi",
+  "MYS": "Malaysia",
+  "NAM": "Namibia",
+  "NER": "Niger",
+  "NGA": "Nigeria",
+  "NIC": "Nicaragua",
+  "NLD": "Netherlands",
+  "NOR": "Norway",
+  "NPL": "Nepal",
+  "NZL": "New Zealand",
+  "OMN": "Oman",
+  "PAK": "Pakistan",
+  "PAN": "Panama",
+  "PER": "Peru",
+  "PHL": "Philippines",
+  "PNG": "Papua New Guinea",
+  "POL": "Poland",
+  "PRK": "North Korea",
+  "PRT": "Portugal",
+  "PRY": "Paraguay",
+  "QAT": "Qatar",
+  "ROU": "Romania",
+  "RUS": "Russia",
+  "RWA": "Rwanda",
+  "SAU": "Saudi Arabia",
+  "SDN": "Sudan",
+  "SEN": "Senegal",
+  "SGP": "Singapore",
+  "SLB": "Solomon Islands",
+  "SLE": "Sierra Leone",
+  "SLV": "El Salvador",
+  "SOM": "Somalia",
+  "SRB": "Serbia",
+  "SSD": "South Sudan",
+  "STP": "Sao Tome and Principe",
+  "SUR": "Suriname",
+  "SVK": "Slovakia",
+  "SVN": "Slovenia",
+  "SWE": "Sweden",
+  "SWZ": "Eswatini",
+  "SYC": "Seychelles",
+  "SYR": "Syria",
+  "TCD": "Chad",
+  "TGO": "Togo",
+  "THA": "Thailand",
+  "TJK": "Tajikistan",
+  "TKM": "Turkmenistan",
+  "TLS": "Timor",
+  "TON": "Tonga",
+  "TTO": "Trinidad and Tobago",
+  "TUN": "Tunisia",
+  "TUR": "Turkey",
+  "TZA": "Tanzania",
+  "UGA": "Uganda",
+  "UKR": "Ukraine",
+  "URY": "Uruguay",
+  "UZB": "Uzbekistan",
+  "VCT": "Saint Vincent and the Grenadines",
+  "VEN": "Venezuela",
+  "VNM": "Vietnam",
+  "VUT": "Vanuatu",
+  "WSM": "Samoa",
+  "YEM": "Yemen",
+  "ZAF": "South Africa",
+  "ZMB": "Zambia",
+  "ZWE": "Zimbabwe"
+}
```

Comment:
If a dataset provides ISO country codes, we don't need to harmonize names. The countries_regions dataset already has those ISO codes, which you could use directly to get the harmonized country names.

Reply:
Thanks @pabloarosado - this has been resolve now.